randovania/games/fusion/logic_database/Sector 5 ARC.txt

```diff
@@ -1267,7 +1295,9 @@ Extra - room_id: [45]
       Any of the following:
           Underwater Wall Jump (Expert)
           Gravity Suit and Space Jump
-          Hi-Jump and Can Freeze Enemies
+          All of the following:
+              Can Freeze Enemies
+              Hi-Jump or Movement (Beginner)
```

Comment:
whats the movement beginner for?

Reply:
you can climb the fish NHJ pretty easily, but i didnt think that method should be trickless

Reply:
Ah gotcha. Please add a comment for that then.

Reply:
please add a comment